recipes/libproperties/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(PackageTest CXX)
```

libproperties is a pure C libary.
Please make the test also pure C 
```suggestion
project(PackageTest C)
```